components/BottomSheet/src/MDCBottomSheetController.h

```diff
@@ -110,6 +110,8 @@
  */
 @property(nonatomic, assign) MDCShadowElevation elevation;
 
+- (nonnull instancetype)init NS_DEPRECATED_IOS(9_0, 9_0, "Use initWithContentViewController:");
```

Comment:
You mention in the PR summary that there's a potential crash. Unless you mark `init` is unavailable, please add guards to `viewDidLoad` to prevent the crash.

Reply:
Updated.